megcruzen/browserify_productOrdering
# Styled products section

# Description
Styled products section and set it up to add inner review section next.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout add-product-styling`
3. Run server/grunt
4. Verify that products section is now styled to fit both desktop and mobile views. 
5. Verify that new div with `review_box` class shows inside product box.

# Checklist:
